### DISCUSSION TITLE
Add an icons for download and import statements in notifications panel

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#15325] [Ideas] [answered]
![ScreenShot_20211028011811](https://user-images.githubusercontent.com/86934170/139155757-234da2d7-9be1-4c0b-95cb-cc001f0de4a7.png)
![ScreenShot_20211028011828](https://user-images.githubusercontent.com/86934170/139155761-4f3c00f2-3f6f-42fb-8f11-6fc6b002ee21.png)

Maybe animated would be better.

**Comment by peppy** [comment, marked as answer]
That's super weird. I swear there used to be one - this might be a regression?

**Reply by peppy** [reply]
Will be addressed via #15338.

**Comment by bdach** [comment]
The screenshots in the opening post look edited. @loreos7 did you edit them? if so, what was the intention for doing so?

**Reply by Flamiii** [reply]
I don't think he edited them as there aren't any icons when downloading/importing for me either. I have no idea if there used to be any icons for that though.

![noicon](https://user-images.githubusercontent.com/74088315/139200669-823b7a53-6ac5-4529-a294-f74799922968.png)

![noicon2](https://user-images.githubusercontent.com/74088315/139200671-8bc7750e-0766-4010-b50c-e4de56a4235c.png)

**Reply by bdach** [reply]
thanks, the red boxes threw me off. i wasn't sure what i was looking at.

**Reply by Loreos7** [reply]
Highlighted places for better understanding but now i see it turned out weird
